src/controllers/user.js

```diff
@@ -83,4 +83,60 @@ const logout = (req, res) => {
   }
 };
 
-module.exports = { login, register, logout };
+const createUser = async (req, res) => {
+  try {
+    const user = await User.create(req.body);
```

Comment:
i think here we need to destruct the req.body since in the frontend we may send other fields not only the fields needed for the user to be created so it can be like this `const {username, name, password, etc} = req.body`

Reply:
I did a mistake here there should be createUser
because the user can not be created without registering 
so I will delete createUser function.

Reply:
i think we can keep it for the admin side in the frontend